src/parsers/manifest/dash/indexes/timeline/timeline_representation_index.ts

```diff
@@ -281,7 +289,12 @@ export default class TimelineRepresentationIndex implements IRepresentationIndex
    * @returns {Object}
    */
   getInitSegment() : ISegment {
-    return getInitSegment(this._index);
```

Comment:
As `getInitSegment` is a DASH specific function, maybe we could factorize the privateInfos thing there